app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/NotesDisplayedActivityTest.kt

```diff
@@ -74,20 +74,20 @@ class NotesDisplayedActivityTest {
         }
     }
 
-    @Test
-    fun playButtonWork() {
-        val uriOfFile = getUriFromAsset(context, "BeetAnGeSample.xml")
-        val intent = Intent(ApplicationProvider.getApplicationContext(), NotesDisplayedActivity::class.java)
-        intent.putExtra("FILE", uriOfFile.toString())
-
-        ActivityScenario.launch<NotesDisplayedActivity>(intent).use{ scenario ->
-            Thread.sleep(1000) //Let time to the player to display the data
-            onView(withId(R.id.play_button)).perform(click())
-            scenario.onActivity {
-                assertTrue(it.isPlaying(),"The player should be playing" )
-            }
-        }
-    }
+//    @Test

```

Comment:
As discussed with the team, this test fails randomly and is causing too much trouble so we remove it for now.